app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,280 @@
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/fmlGuidelinePath10"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                app:layout_constraintGuide_percent="0.08" />
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/fmlGuidelinePath20"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                app:layout_constraintGuide_percent="0.16" />
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/fmlGuidelinePath50"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                app:layout_constraintGuide_percent="0.50" />
+
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/fmlguidelineCenter50"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                app:layout_constraintGuide_percent="0.5" />
+
+
+            <View
+                android:id="@+id/fmlCombinedShape"
+                android:layout_width="16dp"
+                android:layout_height="16dp"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentTop="true"
+                android:layout_marginStart="16dp"
+                android:background="@drawable/combined_shape"
+                app:layout_constraintBottom_toTopOf="@+id/fmlGuidelinePath10"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/fmlGuidelinePath10"
+                tools:ignore="MissingConstraints" />
+
+            <TextView
+                android:id="@+id/fmlWarszawaPo"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentEnd="true"
+                android:layout_centerVertical="true"
+                android:layout_marginStart="8dp"
+                android:gravity="center_vertical"
+                android:lineSpacingExtra="4sp"
+                android:text="@string/fmlWarszawaPo"
+                android:textAppearance="@style/fmlStyleWarszawaPo"
+                app:layout_constraintBottom_toTopOf="@+id/fmlGuidelinePath10"
+                app:layout_constraintStart_toEndOf="@+id/fmlCombinedShape"
+                app:layout_constraintTop_toTopOf="@+id/fmlGuidelinePath10"
+                tools:ignore="MissingConstraints" />
+
+            <View
+                android:id="@+id/fmlBackGroundMovie"
```

Comment:
@kamar19 Please create another ConstaraintLayout for the list item and put all views of this item inside (It will help you in the future). This approach you use is not a very good one.

Reply:
Выполнил, как понял, без списка. Всё в ConstraintLayout.